src/server/graphql/models/User/userMutation.js

```diff
@@ -151,7 +151,13 @@ export default {
         createdAt: new Date(userInfo.created_at),
         tms: userInfo.tms
       };
-      const {email, id: userId, picture, preferredName} = auth0User;
+      const {email, id: userId, createdAt, picture, preferredName} = auth0User;
```

Comment:
Let me borrow my favorite  @mattkrick-ism

Nooooooo....that's not what this call is for 😜 

The call is `updateUserWithAuthToken`, this mutation is called by the client on each login. Where doing this year to make sure a segment event has been called to correlate future server-side segment events with this user.

As for tracking profile updates, we're leaving that up to client-side segment events for now...